web/src/components/dashboard/OpportunityCard.test.tsx

```diff
@@ -85,31 +85,27 @@ describe("OpportunityCard", () => {
     expect(screen.getByText("Rolling Application")).toBeInTheDocument();
   });
 
-  it("displays full first 150 characters of description", () => {
-    const characters = Array(151).fill("a").join("");
-    const expectedTextOnPage = `${Array(150).fill("a").join("")}...`;
+  it("displays OPPORTUNITY_CARD_MAX_BODY_CHARS (minus 3) chars & ellipsis in description when sidebarCardBodyText is OPPORTUNITY_CARD_MAX_BODY_CHARS + 1 length", () => {
```

Comment:
[dust] I think this test name is unnecessarily long.
```suggestion
  it(`displays first `${OPPORTUNITY_CARD_MAX_BODY_CHARS - 3} characters of description & ellipsis`, () => {
```